### ISSUE TITLE
[Traffic Register] Enhancement: Paginated reports updates from usability testing

### ISSUE THREAD

**Issue by ChristinaTremel** [cityofaustin/atd-data-tech#18024]
These items were outlined by users as requested changes on the Paginated Reports:
- [Draft Regulation Document Report](https://app.powerbigov.us/groups/c78e625c-c676-4727-9d5b-b5f0565ee975/rdlreports/ea539be0-9880-4c0d-a40b-f43107d1dc87?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f&openReportSource=ReportInvitation)
- [Regulation Document Report](https://app.powerbigov.us/groups/c78e625c-c676-4727-9d5b-b5f0565ee975/rdlreports/5a54084a-4eed-4f43-b105-350aae331d94?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f&openReportSource=ReportInvitation)

## Action Items:
- [x] For regulation line items, add a date column after the reg text on the Paginated Report for the install/uninstall dates associated to each line item instead of at the top of the page.
- [x] During [usability testing](https://miro.com/app/board/uXjVKY46Xtg=/), users requested that the Knack ID associated to each regulation line item be removed from this document 
![image](https://github.com/cityofaustin/atd-data-tech/assets/61209591/ee00cda6-9875-4c4b-9368-ca736d085441)
- [x] Add City Manager to paginated report document after #18764

**Comment by ChrispinP** [comment]
**Updating PowerBI Data Sources**
PowerBI reports updated and files are stored in the Traffic Register folder on the G Drive. Data was transformed to remove the `<br> , <p> , & <span>` start and closing tags from the Remarks field.

In addition, for the approved data, both TE and CTE api data was merged into a single data table called 'AllMergedData' while for the draft data is just 'DraftData' without signature data.

Both datasets were published to TPW Development workspace

**Updating Paginated Reports**
Since I updated the API tables from Knack and the data sources, I needed to reset to the new data source in each of the paginated reports. After completing the new queries I updated the design to incorporate the adjustments to City Manager, Dates, and removing the Reg Key. For the Install/Uninstall dates in particular, we use a formula to conditionally display the correct date based on if it is an Install or Retire reg.

In Knack I built new fields for each of the paginated reports links that will exist in each of the 3 TPW workspace environments: TPW Development (1st), TPW Test (2nd), and TPW Production (3rd) so we will be able to test the report at each stage. Once we push to Prod for the first time, we can update the backend link formula so that report viewers will see that version of the paginated report.

Here is a sample report:
![image](https://github.com/user-attachments/assets/8c07e6eb-eeb3-4dad-b712-4a12d337a1be)

Complete, pending review & testing

**Comment by ChrispinP** [comment]
The Paginated Report is now published to Prod with the links updated and the button repositioned to the left side of the screen. Some minor color/icon enhancements to make it stand out more for users and look akin to an Adobe PDF style.
![image](https://github.com/user-attachments/assets/73130ebc-2ec5-4d1c-ba13-da3450ac37f4)


Everything except signatures has been tested, records updated, and the Approved dataset is now being refreshed every 10 min with PowerAutomate to help reduce failures due to the dataset being so large.

We provided Viewer access to all Traffic Register accounts to TPW Production workspace so users can now view reports without having to request access individually.

Need to confirm users can access the report via PowerBI.

Also pending signature testing.

**Comment by ChrispinP** [comment]
closing this since we have #16712 for testing signatures
